Static/DataSource.swift

```diff
@@ -188,11 +187,11 @@ extension DataSource: UITableViewDataSource {
     }
 
     public func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
     }
     
     public func tableView(tableView: UITableView, estimatedHeightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
```

Comment:
I don't think we can set this to `UITableViewAutomaticDimension`. It needs to be an explicit height according to
http://blog.jldagon.me/blog/2013/12/07/auto-layout-and-uitableview-cells/

Maybe it we should make `Row.estimatedHeight` a `non-optional` property with default `44`?